api/swagger.yml

```diff
@@ -2541,6 +2546,12 @@ paths:
         required: true
         schema:
           type: string
+      - in: query
+        name: user_metadata
+        required: false
+        schema: 
+          type: boolean
+          default: false
```

Comment:
The default here is false, the next API (ls) the default is set to true - is it on purpose?

Reply:
This should be set to true, thanks for the catch!